package.json

```diff
@@ -26,6 +26,7 @@
     "inquirer": "^6.2.0",
     "mysql": "^2.16.0",
     "read-yaml": "^1.1.0",
+    "shell-escape": "^0.2.0",
```

Comment:
This package hasn't been updated in 5 years or so (likely because it's so simple). It might be worth looking to see if there is something more actively supported, otherwise this is fine and we can always swap it later if needed.

Reply:
@aaemnnosttv if you check this package source code, you will see just one function there. That function does the job, and the fact that it has been used heavily for 5 years without updates convinces me that it is a solid solution. I don't think that we need to look for another solution for it.